k8s-install/README.md

```diff
@@ -2,10 +2,19 @@
 
 This directory includes manifests for deploying canal on Kubernetes using the Kubernetes API.  
 
-**Kubernetes 1.5**:
+**For Kubernetes 1.7**:
+
+> **Note:** If you are upgrading from the Kubernetes 1.6 or 1.5 manifests
+to the 1.7 manifest it is neccessary to
+[migrate your Calico configuration data](https://github.com/projectcalico/calico/blob/master/upgrade/v2.5/README.md)
+before upgrading.  Otherwise, your cluster may lose connectivity after the
+upgrade.  Ensure that after the upgrade you use the Canal RBAC manifest in
+step 4.1 and the Canal manifest in step 4.2 of the migration guide.
 
```

Comment:
I think we should add one more line saying Calico v2.5.0+ only works with k8s v1.7.0+

Reply:
Should be only one space between sentences. Can you also add hyperlinks to the references that you've added here? You can hover over the heading in GitHub and it will show two chain links which allow you to copy the path to the anchor links. I would also recommend hyperlinking "Canal manifest"...I am not sure where that is located. 

However, to be honest, after I have read this sentence several times, I am not sure how to perform this task that you mention. :(

Reply:
@gunjan5 I'm not sure about adding a note about Calico v2.5.0+ only working with K8s v1.7.0+. The docs don't mention what version of Calico are in each of the different manifests so I'm not sure if that will help clarify anything.
We could add to each of the K8s sections the versions of Calico and Flannel that they include and then it might make more sense.  But I don't think the note you're suggesting is needed, is there something you think people will be doing that we need to guard against?